apps/server/src/modules/user/user.module.ts

```diff
@@ -7,14 +7,27 @@ import { RoleModule } from '@src/modules/role/role.module';
 import { RoleService } from '@src/modules/role/service/role.service';
 import { RoleUc } from '@src/modules/role/uc/role.uc';
 import { SchoolModule } from '@src/modules/school/school.module';
+import { AccountModule } from '@src/modules/account';
 import { UserController } from './controller';
 import { UserService } from './service/user.service';
 import { UserUc } from './uc';
+import { AccountService } from '../account/services/account.service';
 
 @Module({
-	imports: [SchoolModule, RoleModule, LoggerModule],
+	imports: [SchoolModule, RoleModule, AccountModule, LoggerModule],
 	controllers: [UserController],
-	providers: [UserRepo, UserDORepo, PermissionService, UserUc, UserService, RoleRepo, RoleUc, SchoolRepo, RoleService],
+	providers: [
+		UserRepo,
+		UserDORepo,
+		PermissionService,
+		UserUc,
+		UserService,
+		RoleRepo,
+		RoleUc,
+		SchoolRepo,
+		RoleService,
+		AccountService,
```

Comment:
AccountService should not be provided by the userModule. use the one in the AccountModule instead.

Reply:
yes, thanks